backend/src/api/middlewares/authenticationMiddleware.ts

```diff
@@ -0,0 +1,26 @@
+import { getCustomRepository } from 'typeorm';
+import { Request, Response, NextFunction } from 'express';
+import { compare } from '../../common/utils/encryptHelper';
+import UserRepository from '../../data/repositories/userRepository';
+
+const authenticationMiddleware = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
You were trying to use Passport so I expected that you would create a strategy. It's not a strategy so you can use kind of code in the service